TypeInjections/Tool/Tool.fsproj

```diff
@@ -3,25 +3,32 @@
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
+
+        <PackAsTool>true</PackAsTool>
+        <ToolCommandName>typecart</ToolCommandName>
+        <PackageOutputPath>./nupkg</PackageOutputPath>
+        <PackageId>typecart</PackageId>
+        <Version>1.0.0</Version>
+        <Authors>syedahir florrabe ruijief schlaipf tschuerl</Authors>
+        <Company>ARI@AWS</Company>
     </PropertyGroup>
 
     <ItemGroup>
-        <Compile Include="Tool.fs" />
+        <Compile Include="Tool.fs"/>
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\TypeInjections\TypeInjections.fsproj" />
+        <ProjectReference Include="..\TypeInjections\TypeInjections.fsproj"/>
```

Comment:
A large part of this diff are these space changes, where does it come from? Some tool?

Reply:
I formatted the file with fantomas